<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAnnotationFullWidget</class>
 <widget class="QWidget" name="CreateAnnotationFullWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create Annotations</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="annotationParametersLayout" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="lblAnnotationType">
         <property name="text">
          <string>Annotation type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwAnnotationType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="spacerLayout" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="parametersLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <property name="spacing">
            <number>11</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="lblAnnotationName">
             <property name="text">
              <string>Annotation name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QCheckBox" name="chbUsePatternNames">
             <property name="toolTip">
              <string>Use names of patterns as annotations names. In case the patterns are in FASTA format</string>
             </property>
             <property name="text">
              <string>Use pattern names</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lblGroupName">
             <property name="text">
              <string>Group name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="tbSelectGroupName">
             <property name="toolTip">
              <string>Predefined group names</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../../../U2Gui.qrc">
               <normaloff>:/core/images/group_auto.png</normaloff>:/core/images/group_auto.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="leAnnotationName">
             <property name="maxLength">
              <number>100</number>
             </property>
             <property name="placeholderText">
              <string>by type</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="leDescription"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblDescription">
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leGroupName">
             <property name="placeholderText">
              <string>auto</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="gbLocation">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Location</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>10</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbSimpleFormat">
                  <property name="text">
                   <string>Simple format</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="gbSimpleFormat">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMinAndMaxSize</enum>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <property name="sizeConstraint">
                    <enum>QLayout::SetMinAndMaxSize</enum>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="leRegionStart"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="lblDash">
                     <property name="text">
                      <string>-</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="leRegionEnd"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="chbComplement">
                   <property name="text">
                    <string>Complement</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>10</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbGenbankFormat">
                  <property name="text">
                   <string>GenBank/EMBL format</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="gbGenbankFormat">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMinAndMaxSize</enum>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="leLocation">
                   <property name="toolTip">
                    <string>Annotation location in GenBank format</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="tbDoComplement">
                   <property name="toolTip">
                    <string>Add/remove complement flag</string>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../U2Gui.qrc">
                     <normaloff>:/core/images/do_complement.png</normaloff>:/core/images/do_complement.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="saveAnnotationsInnerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Choose a file to store the annotations</string>
     </property>
     <layout class="QGridLayout" name="saveAnnotationsLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item row="3" column="0">
       <widget class="QRadioButton" name="rbUseAutoTable">
        <property name="text">
         <string>Use auto-annotations table</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbExistingTable">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="tbBrowseExistingTable">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../U2Gui.qrc">
          <normaloff>:/core/images/load_selected_documents.png</normaloff>:/core/images/load_selected_documents.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leNewTablePath">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbExistingTable">
        <property name="text">
         <string>Existing table</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbCreateNewTable">
        <property name="text">
         <string>Create new table</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="tbBrowseNewTable">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../U2Gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rbExistingTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbExistingTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbExistingTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>tbBrowseExistingTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCreateNewTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>leNewTablePath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCreateNewTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>tbBrowseNewTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSimpleFormat</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbSimpleFormat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbGenbankFormat</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbGenbankFormat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
